src/views/tutorial-view/components/featured-in-collections/helpers.ts

```diff
@@ -1,15 +1,39 @@
-import { Collection as ClientCollection } from 'lib/learn-client/types'
-import { CollectionCardProps } from '../collection-card'
+import { getCollectionSlug } from 'views/collection-view/helpers'
+import {
+  Collection as ClientCollection,
+  themeIsProduct,
+} from 'lib/learn-client/types'
+import { ProductSlug } from 'types/products'
+import { CollectionCardPropsWithId } from './types'
 
 export function formatCollectionCard(
   collection: ClientCollection
-): CollectionCardProps {
+): CollectionCardPropsWithId {
   return {
     id: collection.id,
-    name: collection.name,
-    slug: collection.slug,
     description: collection.description,
-    theme: collection.theme,
-    numTutorials: collection.tutorials.length,
+    heading: collection.name,
+    productsUsed: parseProductsUsed(collection),
+    tutorialCount: collection.tutorials.length,
+    url: getCollectionSlug(collection.slug),
+  }
+}
+
+/**
+ * TODO: Designs show multiple productsUsed on collection cards.
+ *
+ * Should we parse the tutorials within a collection in order
+ * to show a more complete list? Or will this result in more of a
+ * "product icon overload" on these cards?
+ */
+function parseProductsUsed(collection: ClientCollection): ProductSlug[] {
+  const theme = collection.theme
+  if (themeIsProduct(theme)) {
+    // For product themes, return the product slug,
+    // which is compatible with dev-dot's ProductSlug types
+    return [theme]
+  } else {
+    // For other themes, return "hcp", for a generic "H" logo
```

Comment:
Currently in learn if a collection has a non product theme we don't render a logo at all...

Reply:
See the cloud page for an example - https://learn.hashicorp.com/cloud

We can totally render the generic cloud logo, just pointing it out

Reply:
Ah cool, I'll drop this then 👍

Reply:
ah sorry another case of crossed wires - do you think i should add back returning `cloud` here?

Reply:
I think we should just render no logo unless its a valid `product`

Reply:
✅ Sweet, done 👍 Can always adjust later, of course 🎛️